src/main/resources/application.properties

```diff
@@ -8,3 +8,9 @@ spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 
 spring.security.enabled=false
+
```

Comment:
Change secret, add file to gitignore and remove it from PR

Reply:
Done